src/client/pps/pps.proto

```diff
@@ -112,6 +112,9 @@ message ResourceSpec {
   // The amount of memory, in bytes, each worker needs (in bytes, with allowed
   // SI suffixes (M, K, G, Mi, Ki, Gi, etc).
   string memory = 2;
+
+  // The number of GPUs each worker needs.
+  int64 gpu = 3;
```

Comment:
Drive by commenting is always welcome :D, currently it does not support partial GPUs and it's unclear if that's in the cards, I didn't see any indication that it was.